### PR TITLE
Add LinkedIn to social-previews package

### PR DESCRIPTION
**Pull request by manzoorwanijk**
Related to 1203820933396971-as-1204281808721018/f

## Proposed Changes

* Create `LinkedInPreview` component in `social-previews` package
* Use `LinkedInPreview` in WPCOM posts share preview at `https://wordpress.com/posts/:site`

### Prerequisites
- Make sure you have a LinkedIn account.
- Make sure you have a self-hosted site connected to Jetpack, with a subscription that includes Social.
- From the _Social_ or _My Jetpack_ page of your site WPadmin, activate Social.
- Then, connect your site to LinkedIn (check [these instructions](https://jetpack.com/support/jetpack-social/connecting-to-social-networks/) if you need help). You may need to create a test page.
- Spin up Calypso, by running this branch locally or using a live link below.

### Testing

- Create a new post from your site `wp-admin`
- Add a featured image (do not add a custom media)
- Publish it
- Visit `/posts/:site` in Calypso
- You should see the newly created post under the _Published_ tab
- In the post menu, click _Share_, then press the _Preview_ button
- Select the LinkedIn preview
- You should see a preview of how the post looks when shared
Screenshot 2023-04-18 at 2 46 25 PM

When used in Jetpack Social preview (PR coming soon)
Screenshot 2023-04-18 at 1 11 00 PM
Screenshot 2023-04-18 at 1 12 32 PM

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1204281808721018

### CODE REVIEW

File: packages/social-previews/src/linkedin-preview/index.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+				) : null }
+				<article>
+					<img className="linkedin-preview__image" src={ image } alt="" />
+					{ url ? (
+						<div className="linkedin-preview__description">
+							<h2 className="linkedin-preview__description--title">{ title }</h2>
+							<div className="linkedin-preview__description--meta">
+								<span className="linkedin-preview__description--url">{ baseDomain( url ) }</span>
+								<span>•</span>
+								<span>x min read</span>
```
Comment (monsieur-z) [inline packages/social-previews/src/linkedin-preview/index.tsx]: Even though we're mimicking previews, I think we should make texts translatable.

**Review by monsieur-z — APPROVED** [review]
I think texts should be translatable. Except that, everything checks out.

File: packages/social-previews/src/linkedin-preview/index.tsx (hunk 1 above)
Comment (manzoorwanijk) [reply packages/social-previews/src/linkedin-preview/index.tsx]: Done. Thanks

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7933825

Thank you @manzoorwanijk for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7933825).

**Pull request merged**